packages/contentstack-export/src/lib/export/locales.js

```diff
@@ -10,11 +10,6 @@ class LocaleExport {
     this.qs = {
       include_count: true,
       asc: 'updated_at',
-      query: {
```

Comment:
@aman19K This condition was added intentionally to avoid exporting the master locale, and removing this code will affect many places.